client/my-sites/subscribers/components/subscribers-page/subscribers-page-context.tsx

```diff
@@ -77,7 +80,12 @@ export const SubscribersPageProvider = ( {
 	searchTermChanged,
 	sortTermChanged,
 }: SubscribersPageProviderProps ) => {
+	const isDataView = isEnabled( 'subscribers-dataviews' );
 	const [ perPage, setPerPage ] = useState( DEFAULT_PER_PAGE );
+	const [ sortOrder, setSortOrder ] = useState< 'asc' | 'desc' >( 'desc' );
+	const [ dataViewSortTerm, setDataViewSortTerm ] = useState< SubscribersSortBy >(
+		SubscribersSortBy.DateSubscribed
+	);
```

Comment:
Rather than adding these to the query string watcher I just placed this in state